pkg/gofr/datasource/mongo/mongo.go

```diff
@@ -67,8 +68,6 @@ func (c *Client) UseMetrics(metrics interface{}) {
 
 // Connect establishes a connection to MongoDB and registers metrics using the provided configuration when the client was Created.
 func (c *Client) Connect() {
-	c.logger.Logf("connecting to mongoDB at %v to database %v", c.config.URI, c.config.Database)
```

Comment:
Since we have already added the comment to deprecate the URI method to connect to MongoDB , instead we will use the HOST, PORT and PASSWORD fields.

```
// Deprecated Provide Host User Password Port Instead and driver will generate the URI
```